src/MovieView/MovieView.js

```diff
@@ -1,16 +1,27 @@
 import React, { Component } from 'react';
 import ApiCalls from '../ApiCalls.js'
+import Rating from '../Rating/Rating.js';
 import './MovieView.css';
 
 class MovieView extends Component {
   constructor() {
     super();
     this.state = {
-      movie: {}
+      movie: {},
```

Comment:
Great job adding the functionality here to display the user rating & add a new rating. We will need to add functionality to replace an old rating for the user to update their rating on a movie. We will also need to add a condition so that the user cannot rate a movie if they are not signed in. I have created issues for these suggestions.